src/auto/functions.rs

```diff
@@ -237,11 +238,7 @@ pub fn pango_context_get_for_screen(screen: &Screen) -> Option<pango::Context> {
     }
 }
 
-//pub fn pango_layout_get_clip_region(layout: /*Ignored*/&pango::Layout, x_origin: i32, y_origin: i32, index_ranges: i32, n_ranges: i32) -> Option<cairo::Region> {
-//    unsafe { TODO: call ffi::gdk_pango_layout_get_clip_region() }
-//}
-
-//pub fn pango_layout_line_get_clip_region(line: /*Ignored*/&pango::LayoutLine, x_origin: i32, y_origin: i32, index_ranges: &[i32], n_ranges: i32) -> Option<cairo::Region> {
+//pub fn pango_layout_line_get_clip_region(line: &pango::LayoutLine, x_origin: i32, y_origin: i32, index_ranges: &[i32], n_ranges: i32) -> Option<cairo::Region> {
```

Comment:
This function seems also need manual implementation

Reply:
Why is it not in the commented output why a manual impl is needed?

But the `index_ranges` and `n_ranges` need some manual work here. The `n_ranges` is half of the elements in `index_ranges`, and `index_ranges` is `array of byte indexes into the layout, where even members of array are start indexes and odd elements are end indexes.`